app/models/manageiq/providers/amazon/inventory.rb

```diff
@@ -2,4 +2,22 @@ class ManageIQ::Providers::Amazon::Inventory < ManageIQ::Providers::Inventory
   require_nested :Collector
   require_nested :Parser
   require_nested :Persister
+
+  # Default manager for building collector/parser/persister classes
+  # when failed to get class name from refresh target automatically
+  def self.default_manager_name
+    "CloudManager"
+  end
+
+  def self.parser_classes_for(ems, target)
+    case target
+    when ManagerRefresh::TargetCollection
+      [ManageIQ::Providers::Amazon::Inventory::Parser::CloudManager,
+       ManageIQ::Providers::Amazon::Inventory::Parser::NetworkManager,
+       ManageIQ::Providers::Amazon::Inventory::Parser::StorageManager::Ebs]
```

Comment:
I wonder if we could do something generally with child_manager_types on ExtManagementSystem or something...